src/CreeDictionary/CreeDictionary/paradigm/panes.py

```diff
@@ -415,12 +486,12 @@ def contains_wordform(self, wordform: str) -> bool:
         # Must be overridden in subclasses
         raise NotImplementedError
 
-    def fill_one(self, forms: dict[str, Collection[str]]) -> Cell:
+    def fill(self, forms: Mapping[str, Collection[str]]) -> tuple[Cell, ...]:
         """
-        Returns exactly ONE cell by filling the paradigm.
+        Returns one or more cells by filling the paradigm with the provided forms.
         """
         if not self.is_inflection:
-            return self
+            return (self,)
```

Comment:
Why does this need to be a tuple?

Reply:
The API for `Cell.fill()` changed such that filling _any_ cell can return one or more results, hence a tuple. Maybe this isn't the best API change though :/